hack/check_golden_images.sh

```diff
@@ -5,6 +5,17 @@ set -ex
 IMAGES_NS=${IMAGES_NS:-kubevirt-os-images}
 
 if [[ $(${KUBECTL_BINARY} get ssp -n ${INSTALLED_NAMESPACE}) ]]; then
+  # Test image streams
+  [[ $(${KUBECTL_BINARY} get imageStream centos8  -n ${IMAGES_NS} --no-headers | wc -l) -eq 1 ]]
+  [[ "$(${KUBECTL_BINARY} get imageStream centos8  -n ${IMAGES_NS} -o json | jq -cM '.spec.tags[0].from')" == '{"kind":"DockerImage","name":"quay.io/kubevirt/centos8-container-disk-images"}' ]]
+
+  #check that HCO reconciles the image stream
+  ./hack/retry.sh 10 3 "${KUBECTL_BINARY} patch imageStream -n ${IMAGES_NS} centos8 --type=json -p '[{\"op\": \"add\", \"path\": \"/metadata/labels/test-label\", \"value\": \"test\"}]'"
+  sleep 10
+  # HCO expect to remove the test-label label from the image stream
+  ./hack/retry.sh 10 3 "[[ -z $(${KUBECTL_BINARY} get imageStream -n ${IMAGES_NS} centos8 -o jsonpath='{.metadata.labels.test-label}') ]]" "${KUBECTL_BINARY} get imageStream -n ${IMAGES_NS} centos8 -o yaml"
```

Comment:
@nunnatsa  Is this line correct?

Reply:
yes. What do you mean?

Reply:
I understand that retry.sh  is using the last part as debug parameter. there is no issue :+1: